### ISSUE TITLE
Tagless subfolders does not work

### ISSUE THREAD

**Issue by huskelord** [lurkbbs/e621dl#14]
just came back after letting the program run in the background for a bit. it added a my default subfolders where there was no tag aka things like this

[*Fictional]
subfolder = au_ra merfolk

[Species]
subfolder = *Species

didn't work and would end up with a folder for eveything.

I was would it help to make a sub command that makes a serch section not be affected by the default subfolders instead of makeing any search without a tag does'nt get affected by subfolders because this seems like a easier option like below

[Species]
subfolder = *Species
Default subfolders = False

Also if you don't mind could you focus on the blacklist folder thing and the updating post location/deletion thing insead. I ran out to space and i'd rather have those to start getting rid of repeat animations and flash games to save some needed space
especialy since i know i can just do what i did below to work around this problem

[Species/Species1]
subfolder = *Species1

[Species/Species2]
subfolder = *Species2

[Species/Species3]
subfolder = *Species3

[Species/Species4]
subfolder = *Species4

**Comment by lurkbbs** [comment]
just to be clear, you meant

```
[*Fictional]
subfolder = au_ra merfolk

[Species]
subfolder = Fictional  ; Not Species
```

right?

Even better, give me minimal config to reproduce

**Comment by lurkbbs** [comment]
Good day.
Found a great tool to record folder tree structure. Built in Windows, no less.
You just need to open powershell in a folder (Shift + RightClick, open in Powershell) with e621dl.exe and run

`tree .\downloads > tree.txt`

Look at example config and result
[config.ini.txt](https://github.com/lurkbbs/e621dl/files/3957263/config.ini.txt)
[tree.txt](https://github.com/lurkbbs/e621dl/files/3957264/tree.txt)

Also, here folder `taur` has no folders `.everything`, `.overflow` and such, and section `*Taur` has only `subfolder` option. Look at line 1026 and below in `tree.txt` (you can go to line with Ctrl + G in Notepad). I used executable from the latest release.

Edit:
> Also if you don't mind could you focus on the blacklist folder thing and the updating post location/deletion thing insead. I ran out to space and i'd rather have those to start getting rid of repeat animations and flash games to save some needed space
> especialy since i know i can just do what i did below to work around this problem

Oops. But yeah, it''ll be ready no later than Monday 02:00 AM, UTC+3 (aka MSK, FET or TRT)

And for now, to save space quickly, do this:

1. Remove downloads
2. Place all files you want to save in downloads (just multiple offline config runs)
3. Remove all in cache folder
4. Open Powershell in folder with e621dl.exe and run this

`Get-ChildItem -Recurse .\downloads\ | foreach {New-Item -Path .\cache -Name $_.name -ItemType HardLink -Value $_.fullname -ErrorAction SilentlyContinue}`

This should remove everything from cache that you have not in downloads.

Edit 2:
Ah, I see now. You mean there are `images` and `animated` subfolders in species. Yeah, I'll make that option a bit later.

**Comment by lurkbbs** [comment]
Since I finally added folders pruning, both cache and downloads, I think I have some time to fix default sections bug. It is a bug, I found it in my code, and here is how it works.
I check if section has actual search with simple function that checks if there is any whitelisted, blacklisted or anylisted (those tags with `~`), also if there is any condition. So, by default, before this check, I merge blacklisted tags with tags from `[Blacklist]`. And that means if you have it, all you sections have a search. I'll fix it today to more sane solution. I'm sure you'll read it after I upload the new version
